insights/tests/test_foreman_log.py

```diff
@@ -0,0 +1,198 @@
+from insights.tests import context_wrap
```

Comment:
It appears you duplicated this file from `insights/parsers/tests/test_foreman_log.py`.  You should add your tests to the existing file and not add this one.

Reply:
@sagaraivale you should remove this file and add your new tests to the file `insights/parsers/tests/test_foreman_log.py`.  All parser tests should be in the `tests` folder under `parsers`.

Reply:
Please remove this file.

Reply:
@bfahr  I removed ```insights/parsers/tests/test_foreman_log.py```. Kindly review...!